scripts/lib/CIME/build.py

```diff
@@ -260,7 +260,7 @@ def _build_libraries(case, exeroot, sharedpath, caseroot, cimeroot, libroot, lid
             case.set_value("CCSM_CPRNC", os.path.join(full_lib_path, "cprnc"))
             if not os.path.isdir(full_lib_path):
                 os.makedirs(full_lib_path)
-                libs.append("cprnc")
+                libs.insert(0,"cprnc")
```

Comment:
@WesCoomber these are changes to get the cprnc build and test working so I thought you should review.

Reply:
I agree inserting 'cprnc' to the front of the libs list, similar to how we already insert 'mpilib' into libs, makes sense to me.